src/mpid/ch4/generic/mpidig_am_comm_abort.c

```diff
@@ -24,9 +24,7 @@ void MPIDIG_am_comm_abort_init(void)
 /* MPIDIG_COMM_ABORT */
 
 struct am_comm_abort_hdr {
-    int src_rank;
```

Comment:
This information would be helpful in an error or log message (rank %d called MPI_Abort on context_id %x). I suggest keeping it.

Reply:
The point of this PR is now that this structure is internal to this protocol, if you need any additional field, just add it and also add the corresponding code below in the sender and handler to use it. There is zero concern that you need a place holder for potential usage.